confluence-slack-integration/confluence-7-compat/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <artifactId>confluence-slack-integration</artifactId>
         <groupId>com.atlassian.plugins</groupId>
-        <version>1.1.13</version>
+        <version>1.1.14</version>
```

Comment:
I am sorry for the confusion. All changes in `pom.xml` files in all modules need to be reverted. I just got used to thinking about repository modules as about 2 big groups: plugin and common modules. That's why I left just 2 comments - one comment for each group of changes.